pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicPoliciesTest.java

```diff
@@ -2395,4 +2395,31 @@ public void testTopicRetentionPolicySetInManagedLedgerConfig() throws Exception
         });
     }
 
+    @Test
+    public void testPolicyIsDeleteTogether() throws Exception {
```

Comment:
We should add a test for covering the topic auto-deletion case.

Reply:
done.